recipes/mono/mono-windows.recipe

```diff
@@ -25,9 +25,10 @@ class Recipe(recipe.Recipe):
         'bin/zlib1.dll',
         'etc/mono',
         'lib/mono/gac',
-        'lib/mono/2.0',
-        'lib/mono/4.0',
+        'lib/mono/2.0-api',
+        'lib/mono/4.0-api',
         'lib/mono/4.5',
```

Comment:
Yes, it still exists: 

> The 4.5 folder contains the actual assemblies that are used during runtime. We’re considering these the latest assemblies from now on.
> http://www.mono-project.com/docs/about-mono/releases/4.4.0/

Actually in a fresh install there is also the `4.0` folder, but I'm not sure we need that one.